packages/replay-next/src/suspense/AnalysisCache.ts

```diff
@@ -18,6 +18,7 @@ export interface AnalysisParams {
 }
 
 export interface RemoteAnalysisResult {
+  failed?: boolean;
```

Comment:
nit: I don't like optional attributes. We could mark this one non-optional if we assigned like `failed: result.failed === true`